Resources/Prototypes/Reagents/gases.yml

```diff
@@ -245,6 +245,22 @@
   metabolisms:
     Gas:
       effects:
+      - !type:Emote
+        conditions:
+        - !type:ReagentThreshold
+          reagent: NitrousOxide
+          min: 0.2
+        emote: Laugh
+        showInChat: true
+        probability: 0.1
+      - !type:Emote
```

Comment:
Why does this cause screaming, too?

Reply:
I propose screaming as a rarer emote as an additional touch. Nitrous oxide is said to sometimes cause hallucinations.